### PR TITLE
(WIP) Adding the logo for MBR-CGI organization to certificates

### PR DESCRIPTION
**Pull request by halawa**
Asana Task:
      https://app.asana.com/0/60811306289035/173333575055775

### CODE REVIEW

File: lms/djangoapps/edraak_certificates/edraakcertificate.py [hunk 1]
```diff
@@ -54,6 +54,8 @@ def get_organization_logo(organization):
         return 'pmijo.jpg'
     elif organization == 'qou':
         return 'qou.png'
+    elif organization == 'MBRCGI':
```
Comment (OmarIthawi) [inline lms/djangoapps/edraak_certificates/edraakcertificate.py]: Please write the organization name in the lower case. This is needed due to `organization = organization.lower()` line above.

**Comment by OmarIthawi** [comment]
Sherif. There's something wrong with the logo image and it's position. Please check the screenshot bellow:

screen shot 2016-08-31 at 2 26 00 pm

I would always put an image with the same dimension of the other images and put a white background to avoid any possible issues.

As you know the engine (both of Edraak's and ReportLab's parts) is not as smart as browser's HTML/CSS which can handle such differences easily.

**Comment by halawa** [comment]
Done. Thanks :)

**Comment by halawa** [comment]
Okay, can you try again? I hired a better painter :))

**Comment by halawa** [comment]
Actually @OmarIthawi , this still needs a higher quality logo from Amer, so it's not done yet. I'll let you know once it is ready.

**Comment by halawa** [comment]
Okay, @OmarIthawi , wanna give it a try now? Certificate looks good when generated on my local env.

**Comment by OmarIthawi** [comment]
@halawa the certificate looks better. It has some slight issues:

 - The logo is a little big.
 - The position of the logo is not well fit, it looks like eating the whole certificate.

When in doubt, I would check with the marketing team. Could you please upload a certificate to check that?

**Comment by halawa** [comment]
Not liking it, but we had to step out of the optimal here and merge. When we refactor the certificate app, we will make sure the logo sizing and placement maths are improved.

**Pull request merged**